tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
 # By default, we want to run tests for Python 2.7, Python 3.5, and run our
 # flake8 checks.
-envlist = py27,py35,flake8
+envlist = py27,py35,flake8,coveralls
```

Comment:
Here's what I did for apitools:
  https://github.com/google/apitools/blob/master/tox.ini

Looking around, there seems to be a mix of models like that one, and people simply making coverage one of the tox envs, and then simply calling coveralls in the `after_success` section in travis.